components/layout_2020/fragment_tree/box_fragment.rs

```diff
@@ -192,6 +192,35 @@ impl BoxFragment {
         )
     }
 
+    pub fn clip_overflow(&self, containing_block_rect: &PhysicalRect<Au>) -> PhysicalRect<Au> {
```

Comment:
I suggest renaming it to `overflow_clip_rect`, because this function doesn't perform any mutations but simply computes something